src/omen/cogs/wvw_commands.py

```diff
@@ -225,8 +227,8 @@ async def matchup(self, interaction: discord.Interaction):
                     inline=True
                 )
                 embed_obj_value_string += f":{server}_circle: {camp_emoji}x{self.match_data_dict[server]['objectives']['camp']:=2} {tower_emoji}x{self.match_data_dict[server]['objectives']['tower']:=2} {keep_emoji}x{self.match_data_dict[server]['objectives']['keep']:=2} {castle_emoji}x{self.match_data_dict[server]['objectives']['castle']:=2}\n"
-                embed_skirmish_value_string += f":{server}_circle: Points: {self.match_data_dict[server]['skirmish']['total_points']:=2} (+{self.match_data_dict[server]['skirmish']['ppt']:=2} per tick)\n"
-            embed.add_field(name="Current Skirmish", value=embed_skirmish_value_string, inline=False)
+                embed_skirmish_value_string += f":{server}_circle: Points: {self.match_data_dict[server]['skirmish']['skirmish_scores'][list(self.match_data_dict[server]['skirmish']['skirmish_scores'])[-1]]} (+{self.match_data_dict[server]['skirmish']['ppt']:=2} per tick)\n"
+            embed.add_field(name=f"Current Skirmish ({list(self.match_data_dict[server]['skirmish']['skirmish_scores'])[-1]} of 84)", value=embed_skirmish_value_string, inline=False)
```

Comment:
This is a QoL change. It's nice to know what the current skirmish is out of the total number.

Reply:
Any screenshots or example output of this?

Reply:
![Skirmish Count](https://user-images.githubusercontent.com/35041/215297821-db8be036-aabd-4681-a92f-5d717a014fd1.png)

This is what it looks like. I don't have the wvw icons, so those aren't showing up, but it's otherwise right. Just a number of what the current skirmish is. Reset is "1" and the one right before reset is "84".

Reply:
looks good